src/demographics.c

```diff
@@ -74,9 +74,19 @@ void set_up_allocate_work_places( model *model )
 			prob[adx][AGE_WORK_MAP[adx]] = 1.0 - other;
 	}
 
-	// randomly assign a work place networks using the probability map
-	for( pdx = 0; pdx < model->params->n_total; pdx++ )
-		model->population[pdx].work_network = discrete_draw( N_WORK_NETWORKS, prob[model->population[pdx].age_group]);
+    // randomly assign a work place networks using the probability map if not healthcare worker. Otherwise, sets work network
+    // to HOSPITAL_WORK_NETWORK (-1).
+    for( pdx = 0; pdx < model->params->n_total; pdx++ )
+    {
+#if HOSPITAL_ON
```

Comment:
I'd make this a check on the params object rather than a build time one
If a user wants to test quickly results with and without hospital on then a rebuild makes this harder.

Reply:
I've just pushed a commit which changes the switch from a make argument to a parameter in the baseline_parameters.csv file. It is now the last param 'hospital_on' which can be set to either 0 or 1